src/sync/bet-placed.js

```diff
@@ -72,6 +82,14 @@ const pendingBetPlaced = async ({ syncPromises, limit }) => {
             if (foundLog) {
               const bet = parseBet({ log: foundLog });
               await DBHelper.insertBet(bet);
+              // Update event leaderboard investments
```

Comment:
👾  above

Reply:
what? lol

Reply:
its a space invader (https://en.wikipedia.org/wiki/Space_Invaders), ya prob before your time lol
means either add whitespace above, or remove it if there was an extra line